Add docs for rule-based password expiration feature (#4719)
wso2/docs-is: en/includes/guides/user-accounts/account-security/password-validation.md

            </ul>
        </td>
    </tr>
    <tr>
        <td>Expiration (days)</td>
        <td>Passwords of users meeting the criteria expire after this number of days.</td>
    </tr>
</table>

### Password history count

The **Password History Count** feature allows you to specify the number of unique new passwords a user must use before an old password can be reused. This enhances account security by preventing the reuse of old passwords.

![Password History Count]({{base_path}}/assets/img/guides/organization/account-security/password-validation/password-history-count.png){: width="800" style="display: block; margin: 0; border: 0.3px solid lightgrey;"}

To enable this, select the corresponding checkbox and configure the following option.

<table>
    <tr>
        <th>Parameter</th>
        <th>Description</th>
    </tr>
    <tr>
        <td>Password History Count</td>
        <td><code>[Optional]</code> The number of unique passwords that must be set before reusing an old password. <br> <b> Example: </b> If set to <code>3</code>, the user cannot reuse the last three passwords they have set.</td>
    </tr>
</table>

### Password input validation

> The **Password Input Validation** feature enables you to set requirements for password complexity. This includes minimum length and required character types.

Reviewer: ```suggestion
The **Password Input Validation** feature enables you to set password complexity requirements which include minimum password length and required character types.
```
